site/reference/pkg/diff/README.md

```diff
@@ -3,21 +3,96 @@ title: "Diff"
 linkTitle: "diff"
 type: docs
 description: >
-   Diff a local package against upstream
+   Show differences between a local package and upstream
 ---
 <!--mdtogo:Short
-    Diff a local package against upstream
+   Show differences between a local package and upstream
 -->
 
-Diff displays differences between upstream and local packages.
+`diff` displays differences between upstream and local packages.
 
 It can display differences between:
 
-- local package and upstream source version
-- local package and upstream new version
+- The local package and the upstream version which the local package was based on.
+- The local package and the latest version of the upstream package. 
 
-The diff tool can be specified.  By default, the local 'diff' command is used to
-display differences.
+`diff` fetches the versions of a package that are needed, but
+it delegates displaying the differences to a command line diffing tool. By 
+default, the 'diff' command line tool is used, but this can be changed with either the
+`diff-tool` flag or the `KPT_EXTERNAL_DIFF` env variable.
+
+### Synopsis
+<!--mdtogo:Long-->
+```
+kpt pkg diff [PKG_PATH@VERSION] [flags]
+```
+
+#### Args
+
+```
+PKG_PATH:
+  Local package path to compare. diff will fail if the directory doesn't exist, or does not
+  contain a Kptfile. Defaults to the current working directory.
+
+VERSION:
+  A git tag, branch, or commit. Specified after the local_package with @, for 
+  example my-package@master.
+  Defaults to the local package version that was last fetched.
+```
+
+#### Flags
```

Comment:
Including Flags in mdtogo:Long prints them twice in CLI help. Try it out. We usually exclude Flags from mdtogo parsing.

Reply:
I don't think excluding the flags from mdtogo:Long is the right way to address this. We should either rely on the standard cobra output, or fully use the generated content from the site. I created an issue way back on this: https://github.com/GoogleContainerTools/kpt/issues/439